packit/cli/validate_config.py

```diff
@@ -27,18 +31,56 @@
     is_flag=True,
     help="Do not make remote API calls requiring network access.",
 )
+@click.option(
+    "-c",
+    "--config",
+    type=click.Path(exists=True),
+    help="Path to a specific Packit configuration file.",
+)
 @cover_packit_exception
-def validate_config(path_or_url: LocalProject, offline: bool):
+def validate_config(
+    path_or_url: LocalProject,
+    offline: bool,
+    config: Optional[str] = None,
+    debug: Optional[bool] = None,
+):
     """
     Validate PackageConfig.
 
     \b
     - checks missing values
     - checks incorrect types
-    - checks whether monitoring is enabled if 'pull_from_upstream` is used
+    - checks whether monitoring is enabled if 'pull_from_upstream' is used
 
-    PATH_OR_URL argument is a local path or a URL to a git repository with packit configuration file
+    PATH_OR_URL argument is a local path or a URL
+    to a git repository with a packit configuration file.
     """
-    output = PackitAPI.validate_package_config(path_or_url.working_dir, offline)
```

Comment:
Since this was the only use for this method, do you think we can either move the logic you've implemented into that method or remove the method entirely?

I would prefer putting as much as possible into that method so it can be better tested outside of CLI.